docs/spec/loops.md

```diff
@@ -88,7 +88,7 @@ var storageConfigurations = [
 ]
 
 resource storageAccountResources 'Microsoft.Storage/storageAccounts@2019-06-01' = [for (config, i) in storageConfigurations: {
-  name: 'storageAccountNamePrefix${config.suffix}${i}'
+  name: concat(storageAccountNamePrefix,config.suffix,i)
```

Comment:
I think the intention was to use: `'${storageAccountNamePrefix}${config.suffix}${i}'` - could you make that change instead?